server/alertmanager_test.go

```diff
@@ -0,0 +1,66 @@
+package server
+
+import (
+	"testing"
+	"time"
+
+	"github.com/percona/pmm/api/alertmanager/amclient"
+	"github.com/percona/pmm/api/alertmanager/amclient/alert"
+	serverClient "github.com/percona/pmm/api/serverpb/json/client"
+	"github.com/percona/pmm/api/serverpb/json/client/server"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
+)
+
+func TestAlertManager(t *testing.T) {
+	res, err := serverClient.Default.Server.GetSettings(nil)
+	require.NoError(t, err)
+	require.True(t, res.Payload.Settings.TelemetryEnabled)
```

Comment:
And also here, why do you need telemetry enabled?

Reply:
this looks like a copy paste error

Reply:
removed